packages/cli/src/models/config/TruffleConfig.ts

```diff
@@ -24,6 +24,8 @@ const TruffleConfig = {
     if (!networkList[network])
       throw Error(`Given network '${network}' is not defined in your ${this.getTruffleConfigFileName(path)} file`);
     config.network = network;
+    config.networkId = config.network_id;
```

Comment:
>Why do we need networkId and network_id fields on the same object?

We considered deleting `network_id`, but decided to keep it in case it would break something else. I would be okay with deleting it. Would that be better for you?